styles.css

```diff
@@ -399,3 +401,181 @@ span {
+    grid-column: 6 / span 6;
+    justify-content: center;
+  }
+
+  #desktop-contactform {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    background: url(assets/contactbg.PNG);
+    background-repeat: no-repeat;
+    background-size: 100% 90%;
+    width: 100%;
+    padding-bottom: 100px;
+    margin-top: 60px;
+  }
+
+  #desktop-navbar a {
+    color: #3a4a42;
+    margin-left: 15px;
+    font-weight: bold;
+  }
+
+  #projects-card {
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 30px;
+    padding-left: 50px;
+    padding-right: 50px;
+  }
+
+  .project .languages {
+    padding-top: 16px;
+    display: flex;
+    list-style: none;
+    justify-content: center;
+    gap: 12px;
+  }
+
+  #about-myself p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #42526e;
+    padding-left: 220px;
+    padding-right: 220px;
+    box-sizing: border-box;
+  }
+
+  #portfolio-skills {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-auto-rows: minmax(300px, auto);
+    row-gap: 30px;
+    justify-content: center;
+    column-gap: 10px;
+    margin-left: 80px;
+    margin-right: 80px;
+  }
+
+  #headline-section {
+    background-image: url("./assets/Header-llustration-desktop.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: auto;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    width: 100%;
+    margin-top: -40px;
+    padding-bottom: 300px;
+  }
+
+  #headline-section p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    text-align: center;
+    color: #172b4d;
+    padding: 10px;
+    margin-left: 20%;
+    margin-right: 20%;
+    line-height: 132%;
+  }
+
+  #desktop-contactform p {
+    margin-top: 40px;
+    grid-column: 1 /span 6;
+    justify-content: center;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    text-align: center;
+    color: #172b4d;
+    padding: 10px;
+    margin-left: 30%;
+    line-height: 132%;
+  }
+
+  #mobile-navbar img {
+    display: none;
+  }
+
+  #line hr {
+    display: none;
+  }
+
+  #footer-line hr {
+    display: none;
+  }
+
+  #contact-form {
+    display: none;
+  }
+
+  #desktop-contactform .inline-form {
+    grid-column: span 3;
+  }
+
+  .form-inline {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+  }
+
+  form input {
+    padding: 10px;
+    flex-direction: column;
+    padding-right: 300px;
+    margin-left: auto;
+    margin-right: auto;
+    color: #42526e;
+  }
+
+  form textarea {
+    padding: 10px;
+    padding-right: 240px;
+    margin-left: auto;
+    margin-right: auto;
+    color: #42526e;
+    border: 1px solid #60c095;
+    border-radius: 4px;
+  }
```

Comment:
_Excellent job for improving the contact section. Well done 👏👏_
- [ ] Good job for fixing the background image 🥇🥇 I noticed that two input fields are present for the name. But in the previous review, only one field was present (for name) 📌Also, the `textarea` was almost aligned with the other fields. Kindly make sure to remove the field for tha last name as it was present previously. This is also mentioned in the project requirements 📁
- [ ] Also, kindly align the `textarea` with other fields and adjust it's height as well according to how it was present in the last review ⛏🤓

Your design previously | Your design now
---| ---|
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/106119105/ae31340a-d4d7-4dbe-8ee4-7172f9f1c212) | ![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/106119105/2fd28043-a3bd-4d34-8b21-796b90461708)